src/extension/src/ActionHandler.py

```diff
@@ -57,9 +57,28 @@ def determine_operation(self, command):
             raise e
 
     def install(self):
-        self.logger.log("Extension installation started")
-        install_command_handler = InstallCommandHandler(self.logger, self.ext_env_handler)
-        return install_command_handler.execute_handler_action()
+        stdout_file_mirror = None
+        file_logger = None
+
+        try:
+            log_file_name = datetime.datetime.utcnow().strftime("%Y-%m-%d_%H-%M-%S") + "_Install"
+            file_logger = self.utility.create_log_file(self.ext_env_handler.log_folder, log_file_name)
+            if file_logger is not None:
+                self.logger.file_logger = file_logger
+                stdout_file_mirror = StdOutFileMirror(file_logger)
```

Comment:
This chunk of code has been duplicated several times here. Abstract into a function  and reference from all sections that use the exact same pattern.